library/src/main/java/com/novoda/downloadmanager/lib/OkHttpNotificationImageRetriever.java

```diff
@@ -36,4 +49,14 @@ public Bitmap retrieveImage(String imageUrl) {
             return null;
         }
     }
+
+    private void cleanupOldBitmap() {
+        if (bitmap == null) {
+            return;
+        }
+        if (!bitmap.isRecycled()) {
+            bitmap.recycle();
```

Comment:
yes - let the client deal with it (i.e. when the client asks for a new image then they can recycle the old bitmap if they really want)